doc/intro.md

```diff
@@ -5,11 +5,11 @@
 -L: List of supported IO plugins
 -q: Exit after processing commands
 -w: Write mode enabled
--i: Interprets a r2 script
+-i [file]: Interprets a r2 script
 -A: Analyze executable at load time (xrefs, etc)
 -n: Bare load. Do not load executable info as the entrypoint
--c'cmds': Run r2 and execute commands (eg: r2 -wqc'wx 3c @ main')
--p: Creates a project for the file being analyzed (CC add a comment when opening a file as a project)
+-c 'cmds': Run r2 and execute commands (eg: r2 -wqc'wx 3c @ main')
+-p [prj]: Creates a project for the file being analyzed (CC add a comment when opening a file as a project)
```

Comment:
Yes indeed, my intentions weren't to add more command line flags, but, as you can see, to add a missed spacing and the arguments after -i & -p (file, project file). And of course fixing the spelling mistakes. Very small changes :)